### PR TITLE
News feed impl.

### PR DESCRIPTION
**Pull request by arihilmir**

### CODE REVIEW

File: WebApplication/Controllers/HomeController.cs [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
 namespace WebApplication.Controllers
 {
     public class HomeController : Controller
     {
-        private readonly IMapper _mapper;
+        private IHostEnvironment _host;
         private readonly INewsFeedService _newsFeed;
-        private readonly ILogger<HomeController> _logger;
 
-        public HomeController(IMapper mapper, INewsFeedService newsFeed, ILogger<HomeController> logger)
```
Comment (OlegAndrus12) [inline WebApplication/Controllers/HomeController.cs:16]: We refused to use logger?

File: WebApplication/Controllers/HomeController.cs [hunk 2]
```diff
@@ -1,28 +1,48 @@
...
+        public HomeController(INewsFeedService newsFeed, IHostEnvironment host)
         {
-            _mapper = mapper;
+            _host = host;
             _newsFeed = newsFeed;
-            _logger = logger;
         }
 
-        public IActionResult Index()
+        private string EncodeImage(string path)
```
Comment (OlegAndrus12) [inline WebApplication/Controllers/HomeController.cs]: I guess we need such method in any page, where we works with images.
Maybe make it public and static?

**Review by OlegAndrus12 — CHANGES_REQUESTED** [review]

File: WebApplication/Controllers/HomeController.cs (hunk 2 above)
Comment (nikilark) [inline WebApplication/Controllers/HomeController.cs]: I am not sure that encoding images is work for home controller. We could add method like this somewhere in admin page part to upload photos to database. And for now we can use some online services to convert images to base64

File: WebApplication/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+
+@{
     ViewData["Title"] = "Home Page";
 }
 
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-</div>
+@foreach (var n in Model)
```
Comment (nikilark) [inline WebApplication/Views/Home/Index.cshtml]: please replace "n" with something more meaningful

File: WebApplication/Controllers/HomeController.cs (hunk 2 above)
Comment (arihilmir) [reply WebApplication/Controllers/HomeController.cs]: We can move it to Core as ImageHelper.

File: WebApplication/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -1,8 +1,41 @@
...
+@{
     ViewData["Title"] = "Home Page";
 }
 
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-</div>
+@section Styles {
+    <link rel="stylesheet" href="~/css/local.css" />
```
Comment (nikilark) [inline WebApplication/Views/Home/Index.cshtml]: local.css?
maybe use something more like "newsPageStyles.css"?

File: WebApplication/Controllers/HomeController.cs (hunk 1 above)
Comment (arihilmir) [reply WebApplication/Controllers/HomeController.cs:16]: Well, we don't use it here, so it can be safely removed from here.

**Review by nikilark — APPROVED** [review]

**Review by OlegAndrus12 — APPROVED** [review]

**Pull request merged**